test/runtests.jl

```diff
@@ -10,6 +10,9 @@ using FilePathsBase: /, join
 using FilePathsBase.TestPaths
 using UUIDs: uuid4
 using JSON3
+using Mocking
```

Comment:
Would be nice to order these alphabetically

Reply:
yeah I was thinking of doing that but it felt kind of out not related to this particular MR, but if we're fine with it I'll reorder all the usings.